### ISSUE TITLE
PDF link returns 404

### ISSUE THREAD

**Issue by dsuch** [pybind/pybind11#2603]
Hello,

I am just reporting that the link to the PDF version of the documentation currently returns HTTP 404.

https://media.readthedocs.org/pdf/pybind11/master/pybind11.pdf

Regards.

**Comment by bstaletic** [comment]
Wrong link, from a wrong host/organization.

https://pybind11.readthedocs.io/en/stable/
https://pybind11.readthedocs.io/_/downloads/en/stable/pdf/

**Comment by dsuch** [comment]
That wrong link is provided in the documentation and on GitHub. 

https://github.com/pybind/pybind11/blob/01453b05c49a5796fa87d7c1d90931b33beb42a4/README.rst#L52

**Comment by dsuch** [comment]
Interestingly enough, the PDF document that you linked to - for which I thank you - also has a wrong link to itself, i.e. https://buildmedia.readthedocs.org/media/pdf/pybind11/master/pybind11.pdf

**Comment by YannickJadoul** [comment]
Thanks for noticing and taking the time to report, @dsuch. Do you have time to make a PR?

**Comment by YannickJadoul** [comment]
I guess RTD updated their API/links at some point? From that perspective, having this manual link to the PDF might not be the smartest move, but then again, it is quite handy.

**Comment by dsuch** [comment]
Hi @YannickJadoul - I can do it, but is README.rst the only place to update?

**Comment by bstaletic** [comment]
As far as I can tell, it is.

```
bstaletic@Gallifrey pybind11  (git)-[tags/v2.6.0rc3]-% rg media.readthedocs
README.rst
52:`here <https://media.readthedocs.org/pdf/pybind11/master/pybind11.pdf>`_.

docs/conf.py
138:            "//media.readthedocs.org/css/sphinx_rtd_theme.css",
139:            "//media.readthedocs.org/css/readthedocs-doc-embed.css",
```

**Comment by YannickJadoul** [comment]
@dsuch, great!

Yes, since recently, `README.rst` gets included into the docs, such that the same information isn't repeated multiple times :-)

**Comment by henryiii** [comment]
Wow, I didn't realize the PDF build was working, I had that filed away for fixing later. :)

**Comment by henryiii** [comment]
We need to work on the table of contents, though...

```
1 Core features 3
2 Goodies 5
3 Supported compilers 7
4 About 9
Bibliography 157
Index 159
```

**Comment by henryiii** [comment]
Are you going to make a PR @dsuch ?

By the way, the build does have errors with some characters, but is forced to complete; that's why it fails locally but "passes" on CI. Maybe for 1.6.1 we can touch up the PDF a bit.

**Comment by dsuch** [comment]
Yes, here it is https://github.com/pybind/pybind11/pull/2605

As to the table of contents, you are right, it does not look perfect and there is no other way to get started but to rather carefully read through the changelog and migration notes before one arrives at an unassuming chapter "4.2.3 Installing the library", on page 38.

**Comment by YannickJadoul** [comment]
Thanks a lot, @dsuch! :-)

**Comment by YannickJadoul** [comment]
Fixed by #2603
